examples/adjoints/adjoints_ex6/femparameters.h

```diff
@@ -0,0 +1,149 @@
+
+#ifndef __fem_parameters_h__
+#define __fem_parameters_h__
```

Comment:
We don't use preprocessor macros that begin with a double underscore, they are technically reserved for use by the compiler. Use `FEMPARAMETERS_H` instead.